app/components/candidate_interface/application_status_tag_component.html.erb

```diff
@@ -22,22 +22,16 @@
       classes: 'govuk-!-margin-bottom-0 govuk-!-margin-top-2',
     ) do %>
       <p>
-        Some providers allow you to defer your offer. This means that you could start your course a year later.
+        You can defer your offer which means that you could start your course a year later.
       </p>
 
       <p>
-        Every provider is different, so it may or may not be possible to do this.
-        Find out by contacting <%= @application_choice.current_course_option.course.provider.name %>.
+        Contact <%= @application_choice.current_course_option.course.provider.name %> to ask if it’s possible to defer, this will not affect your existing offer.
       </p>
 
       <p>
-        Asking if it’s possible to defer will not affect your existing offer.
+      If your provider agrees, you’ll need to accept the offer on your account first and then defer it.
```

Comment:
Could we drop "on your account"? I'm also not entirely sure if the candidate has to defer it in the interface after accepting, or if they need to get the provider to do it (I probably ought to know this though!)?

```suggestion
      If your provider agrees, you’ll need to accept the offer first and then defer it.
```

Reply:
Good spot, I checked and you're correct. Providers need to do it so I've updated that!